multicluster/transport.go

```diff
@@ -60,12 +71,18 @@ func NewTransport(rt http.RoundTripper) *Transport {
 // NewTransportWrapper create a WrapperFunc for wrapping RoundTripper with
 // multi-cluster transport
 func NewTransportWrapper(options ...TransportOption) transport.WrapperFunc {
-	return func(rt http.RoundTripper) http.RoundTripper {
-		t := NewTransport(rt)
+	return func(rt http.RoundTripper) (ret http.RoundTripper) {
+		t, ok := asTransport(rt)
+		if ok {
+			ret = rt
+		} else {
+			t = NewTransport(rt)
+			ret = t
+		}
```

Comment:
t is either not used(if ok) or covered(if not ok). I did't get why we need return t here?

Reply:
I got it!